### PR TITLE
sentinel middleware doc 

### LINKED ISSUE #1437: [CONTENT] doc for sentinel middleware
**What content needs to be created or modified?**

Doc for sentinel middleware support. The corresponding PR to support sentinel as Dapr's middleware is: https://github.com/dapr/components-contrib/pull/829

**Describe the solution you'd like**

**Where should the new material be placed?**

**Additional context**

### PR DESCRIPTION
**Pull request by beiwei30**
Thank you for helping make the Dapr documentation better!

**Please follow this checklist before submitting:**

- [x] [Read the contribution guide](https://docs.dapr.io/contributing/contributing-docs/)
- [x] Commands include options for Linux, MacOS, and Windows within codetabs
- [x] New file and folder names are globally unique
- [x] Page references use shortcodes instead of markdown or URL links
- [x] Images use HTML style and have alternative text
- [x] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Description

The doc for sentinel middleware (https://github.com/dapr/components-contrib/pull/829) support

## Issue reference

 this PR will close: #1437

### CODE REVIEW

File: daprdocs/content/en/developing-applications/middleware/supported-middleware/middleware-sentinel.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
+---
+type: docs
+title: "Sentinel"
```
Comment (AaronCrawfis) [inline daprdocs/content/en/developing-applications/middleware/supported-middleware/middleware-sentinel.md]:
```suggestion
title: "Sentinel fault-tolerance middleware component"
```

File: daprdocs/content/en/developing-applications/middleware/supported-middleware/middleware-sentinel.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+        }
+      ]
+```
+
+## Spec metadata fields
+
+| Field       | Details                                                                                                                                                                              | Example |
+|----------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------|
+| appName | the name of current running service | `nodeapp`    |
+| logDir | the log directory path | `/var/tmp/sentinel` |
```
Comment (AaronCrawfis) [inline daprdocs/content/en/developing-applications/middleware/supported-middleware/middleware-sentinel.md:47]: Does this work on Kubernetes?

**Review by AaronCrawfis — APPROVED** [review]

**Review by AaronCrawfis — CHANGES_REQUESTED** [review]
Please see comments on title and on Kubernetes question.

Thanks for contributing!

**Comment by AaronCrawfis** [comment]
Pending https://github.com/dapr/components-contrib/pull/829 merge

File: daprdocs/content/en/developing-applications/middleware/supported-middleware/middleware-sentinel.md (hunk 2 above)
Comment (beiwei30) [reply daprdocs/content/en/developing-applications/middleware/supported-middleware/middleware-sentinel.md:47]: yes, it's a general solution and is supposed to work in k8s environment too.

File: daprdocs/content/en/developing-applications/middleware/supported-middleware/middleware-sentinel.md (hunk 1 above)
Comment (beiwei30) [reply daprdocs/content/en/developing-applications/middleware/supported-middleware/middleware-sentinel.md]: done.

File: daprdocs/content/en/developing-applications/middleware/supported-middleware/middleware-sentinel.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
+---
+type: docs
+title: "Sentinel"
+linkTitle: "Sentinel fault-tolerance middleware component"
```
Comment (AaronCrawfis) [inline daprdocs/content/en/developing-applications/middleware/supported-middleware/middleware-sentinel.md]:
```suggestion
linkTitle: "Sentinel"
```

**Review by AaronCrawfis — APPROVED** [review]

**Pull request merged**